### PR TITLE
feat(plugin): SpeechBubble

### PR DESCRIPTION
**Pull request by Rapougnac**
Basically, it adds a way to put speech-bubbles on top of the provided image

### CODE REVIEW

File: src/webpack/common.tsx [hunk 1]
```diff
@@ -64,6 +65,9 @@ export let RelationshipStore: Stores.RelationshipStore & {
     getSince(userId: string): string;
 };
 
+export const fetchUser = findByCodeLazy(".USER(");
```
Comment (Vendicated) [inline src/webpack/common.tsx:68]: could you please update other plugins using these modules (petpet for example)

File: src/plugins/speechBubble.ts [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+                    throw "Failed to fetch user. Check the console for more info.";
+                }
+        }
+    }
+    return null;
+}
+
+export default definePlugin({
+    name: "SpeechBubble",
+    description: "Adds a speech bubble in the top of an image",
```
Comment (Vendicated) [inline src/plugins/speechBubble.ts:84]:
```suggestion
    description: "Slash command to create speechbubble memes",
```

**Comment by Vendicated** [comment]
Sorry for only looking at this now!
Why not make the speechbubble part transparent?
It would also be cool to have the option to add it on the bottom, similar to esmBot

**Comment by Rapougnac** [comment]
> Why not make the speechbubble part transparent?
 It's on my plan, I just got busy last week, but I'll do this, sure

**Pull request closed without merging**